decidim-core/app/cells/decidim/address/online.erb

```diff
@@ -2,8 +2,10 @@
   <%= icon "map-pin-line" %>
   <div class="address">
     <div class="address__location"><%= t(model.type_of_meeting, scope: "decidim.meetings.meetings.filters.type_values") %></div>
-    <a href="<%= model.online_meeting_url %>" target="_blank" rel="noopener noreferrer" class="address__hints underline break-all">
-      <%= model.online_meeting_url %>
+      <% if model.online? && model.iframe_access_level_allowed_for_user?(current_user) %>
```

Comment:
Seems like this doesn't work well, as it's failing in the `decidim-design` specs. 

I think the cleaner way would be to:

1. Move this logic to a method in the cell 
2. Apply some guard clauses in case the model doesn't have these methods
3. Add specs for these cases in `address_cell_spec.rb` 

Let's have a meeting tomorrow to do a bit of pair programming to tackle this